price-feeder/oracle/oracle.go

```diff
@@ -36,7 +36,7 @@ const (
 
 // deviationThreshold defines how many 𝜎 a provider can be away from the mean
 // without being considered faulty.
-var deviationThreshold = sdk.MustNewDecFromStr("2")
+var deviationThreshold = sdk.MustNewDecFromStr("1.0")
```

Comment:
Looking at [this guy](https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule)
We updated the deviation to 1 to get only 68.27% of the possible prices discrepancies